python/prairielearn.py

```diff
@@ -52,6 +54,74 @@ class QuestionData(TypedDict):
 class ElementTestData(QuestionData):
     test_type: Literal['correct', 'incorrect', 'invalid']
 
+class DisplayType(Enum):
+    INLINE = "inline"
+    BLOCK = "block"
+
+def grade_question_parameterized(data: QuestionData,
```

Comment:
Is this used outside of this new element? If not, let's keep it there for now - I want to avoid adding things to this public API unless there's value in doing so.

Reply:
Ah I did this by mistake. This can actually be used in an arbitrary element for grading (including some of the others we have in PR, which is why this is here). But I'll wait to move this here as part of those later PRs.